ecl/eclplus/DumpHelper.cpp

```diff
@@ -147,3 +147,68 @@ bool GraphHelper::doit(FILE * fp)
 }
 
 
+
+
+StatsHelper::StatsHelper(IProperties * _globals, IFormatType * _format) : globals(_globals), format(_format), wuclient(createWorkunitsClient(_globals))
+{
+}
+
+bool StatsHelper::doit(FILE * fp)
+{
+    Owned<IClientWUGetStatsRequest> req = wuclient->createWUGetStatsRequest();
+    const char* wuid = globals->queryProp("WUID");
+    const char* filter = globals->queryProp("FILTER");
+    if (wuid)
+        req->setWUID(wuid);
+    else
+        req->setWUID("*");
+
+    if (filter)
+        req->setFilter(filter);
+
+    Owned<IClientWUGetStatsResponse> resp = wuclient->WUGetStats(req);
+    const IMultiException* excep = &resp->getExceptions();
```

Comment:
Yes it looks like the esp created interfaces are called getXXX but do not link their return value